### ISSUE TITLE
What is the configuration/specs of system that you use for training ?

### ISSUE THREAD

**Issue by pawan9** [cavalleria/cavaface.pytorch#15]
Can you please tell what are the specs of the system that you use for training models and also how time does it take to train a fresh MobileFaceNet, IR-SE-100 model on ms1m-retinanet dataset ?

**Comment by cavalleria** [comment]
Thx for your question. It takes about 8 hours to train mobilefacenet 24 epoch with 8 V100 GPUS, and 27 hours for IR-SE-100.

**Comment by pawan9** [comment]
@cavalleria thanks i really appreciate your work and your quick response

one more question, what is your Training Loss, Training Prec@1 and Training Prec@5 on ms1m-retinanet dataset for MobileFaceNet and IR-SE-100 And for how many epoches do you train them ?

**Comment by cavalleria** [comment]
thx, i check the training logs.
For  MobileFaceNet, Training Loss 5.6511 (5.7445)   Training Prec@1 48.438 (44.848) Training Prec@5 58.008 (56.868)
For IR-SE-100, Training Loss 1.9620 (0.9541)   Training Prec@1 84.375 (88.590) Training Prec@5 89.062 (93.178)
By the way, the validation set acc and megface/ijbc test acc is more important and informative
